[2023-02-19 23:03:43,571] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: members_delta_hourly_extract.validate_task manual__2023-02-19T23:03:32.851323+00:00 [queued]>
[2023-02-19 23:03:43,596] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: members_delta_hourly_extract.validate_task manual__2023-02-19T23:03:32.851323+00:00 [queued]>
[2023-02-19 23:03:43,599] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-02-19 23:03:43,602] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-02-19 23:03:43,605] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-02-19 23:03:43,632] {taskinstance.py:1377} INFO - Executing <Task(MembersDeltaHourlyValidateOperator): validate_task> on 2023-02-19 23:03:32.851323+00:00
[2023-02-19 23:03:43,672] {standard_task_runner.py:52} INFO - Started process 968 to run task
[2023-02-19 23:03:43,682] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'members_delta_hourly_extract', 'validate_task', 'manual__2023-02-19T23:03:32.851323+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/members_delta_extract_hourly_dag/members_delta_hourly_extract.py', '--cfg-path', '/tmp/tmpk4_j1ajr', '--error-file', '/tmp/tmp6hzqkb3m']
[2023-02-19 23:03:43,687] {standard_task_runner.py:80} INFO - Job 61: Subtask validate_task
[2023-02-19 23:03:43,799] {task_command.py:369} INFO - Running <TaskInstance: members_delta_hourly_extract.validate_task manual__2023-02-19T23:03:32.851323+00:00 [running]> on host ff325d103691
[2023-02-19 23:03:43,926] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=members_delta_hourly_extract
AIRFLOW_CTX_TASK_ID=validate_task
AIRFLOW_CTX_EXECUTION_DATE=2023-02-19T23:03:32.851323+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-19T23:03:32.851323+00:00
[2023-02-19 23:03:43,934] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-02-19 23:03:43,989] {members_delta_hourly_validate.py:30} INFO - There are files to validate
[2023-02-19 23:03:43,993] {members_delta_hourly_validate.py:65} INFO - Start Validating Data
[2023-02-19 23:03:44,012] {members_delta_hourly_validate.py:91} INFO - All validation checks have passed
[2023-02-19 23:03:44,036] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=members_delta_hourly_extract, task_id=validate_task, execution_date=20230219T230332, start_date=20230219T230343, end_date=20230219T230344
[2023-02-19 23:03:44,097] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-19 23:03:44,168] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
